operator/pkg/labels/labels.go

```diff
@@ -30,6 +30,11 @@ const (
 	PartOfKey = "app.kubernetes.io/part-of"
 	// The tool being used to manage the operation of an application
 	ManagedByKey = "app.kubernetes.io/managed-by"
+	// NodePoolKey is used to document the node pool associated with the StatefulSet.
+	NodePoolKey = "cluster.redpanda.com/nodepool"
+
+	// PodLabelNodeIDKey
```

Comment:
nit: incomplete comment. Seems like `PodNodeIDKey` would be more consistent with the naming convention here? `Label` is already implied by the package.

Reply:
done